### PR TITLE
Cindy calendar

### PR DESCRIPTION
**Pull request by cindy1u0**
This PR includes integrating airTable to calendar events and also deleting the `fake_data json`. I made a table with all the events (very rough draft might need to adjust later) and incorporated with the schedule we have:
Screen Shot 2021-01-20 at 10 50 01 PM
Screen Shot 2021-01-20 at 10 49 58 PM
Screen Shot 2021-01-20 at 10 49 54 PM

### CODE REVIEW

File: src/components/Schedule.js [hunk 1]
```diff
@@ -14,84 +26,90 @@ const Schedule = () => {
...
-                        );
-                      })}
-                    </tbody>
-                  </Table>
-                </Tab>
-              );
-            })}
-          </Tabs>
-        </Col>
-      </Row>
```
Comment (cindy1u0) [inline src/components/Schedule.js:94]: I have some changes here but idk why it just reformatted everything...

Comment (ghost) [reply src/components/Schedule.js:94]: u can view with ignore whitespace changes! in the little gear icon

**Review by ghost — APPROVED** [review]
Yay

File: src/components/Schedule.js (hunk 1 above)
Comment (cindy1u0) [reply src/components/Schedule.js:94]: wait where is the gear icon?

Comment (ghost) [reply src/components/Schedule.js:94]: Screen Shot 2021-01-21 at 4 37 57 PM

Comment (cindy1u0) [reply src/components/Schedule.js:94]: ooooo

Comment (ghost) [reply src/components/Schedule.js:94]: yeee!

**Review by snwessel — APPROVED** [review]
This looks great!! Thank you for working on it 😀 

I think it should give you permisson to merge this time, but if not let me know

**Comment by cindy1u0** [comment]
> This looks great!! Thank you for working on it 😀
> 
> I think it should give you permisson to merge this time, but if not let me know

I still can't ;-; it says im not authorized ;-;

**Pull request merged**